SEED-platform/seed
# 1647 add cycle to view

#### Any background context you want to provide?
#1647 

#### What's this PR do?
Add the current cycle to multiple views.

#### How should this be manually tested?
Import any file and check for cycle name to be presented during the following 5 views:
<img width="633" alt="Screen Shot 2020-09-17 at 10 45 45 AM" src="https://user-images.githubusercontent.com/49968203/93548430-6fea8200-f924-11ea-8a39-605dba7dc7b1.png">
<img width="821" alt="Screen Shot 2020-09-17 at 10 50 30 AM" src="https://user-images.githubusercontent.com/49968203/93548450-7842bd00-f924-11ea-884a-d321ce0bd42d.png">
<img width="536" alt="Screen Shot 2020-09-17 at 10 51 58 AM" src="https://user-images.githubusercontent.com/49968203/93548483-84c71580-f924-11ea-9eb4-03ae20be7d63.png">
<img width="1171" alt="Screen Shot 2020-09-17 at 10 54 45 AM" src="https://user-images.githubusercontent.com/49968203/93548498-8abcf680-f924-11ea-8dcb-2b60d290c3a0.png">
<img width="632" alt="Screen Shot 2020-09-17 at 10 55 37 AM" src="https://user-images.githubusercontent.com/49968203/93548503-8d1f5080-f924-11ea-9abd-0302e5e1653d.png">

#### What are the relevant tickets?
#### Screenshots (if appropriate)
